src/Resources/TokenResource.php

```diff
@@ -131,6 +131,11 @@ public static function getPages(): array
         ];
     }
 
+    public static function getCluster(): ?string
+    {
+        return config('api-service.navigation.token.cluster') ?? config('api-service.navigation.cluster.token');
```

Comment:
please change to `return config('api-service.navigation.token.cluster')`

you don't need to follow it from `getNavigationGroup` method